test/functionalTest/cases/117_convop_using_standard_ops/postAllEntitiesWithTypeAndId.test

```diff
@@ -672,4 +672,4 @@ Date: REGEX(.*)
 
 --TEARDOWN--
 brokerStop CB
-# dbDrop CB
+dbDrop CB
```

Comment:
What I really plan to do is to add an option to testHarness.sh: --noTeardown

I comment the dbDrop line to be able to see what's in mongo (of course) every time I have doubt about some func-test and it is really easy to forget that commented line in a PR.
With --noTeardown I will no longer need to comment out the call to dbDrop and the problem is solved.

Of course, the search for old '#dbDrop' must still be done ...